server/sonar-server/src/main/java/org/sonar/server/component/ws/FilterParser.java

```diff
@@ -112,6 +136,10 @@ public String getValue() {
       return value;
     }
 
+    public List<String> getValues() {
```

Comment:
what happens if I call getValues for a Criterion that is supposed to have only one value?